src/App/components/ActionList/index.js

```diff
@@ -5,11 +5,11 @@ import { Icon } from "#";
 
 const ActionList = ({ id, classNames, toggleBtn, items }) => (
     <>
-        <div id={id} className={`action-list${classNames ? ` ${classNames}` : ""}`}>{"\n"}
+        <div id={id} className="action-list">{"\n"}
             {toggleBtn ? <>{toggleBtn}{"\n"}</> : <>{<i className="material-icons action-toggle">more_vert</i>}</>}
             <div className={`action-menu${classNames ? ` ${classNames}` : ""}`}>{"\n"}
                 {
-                    (Array.isArray(items)) ? items.map(({ name, icon }, i) => (
+                    items && Array.isArray(items) ? items.map(({ name, icon }, i) => (
```

Comment:
A bit redundant but can't be too careful i guess

Reply:
![image](https://user-images.githubusercontent.com/9350120/58937419-ac678e00-8772-11e9-94c3-c829917acc51.png)
Yeah, you're right. I'll drop it.